gameManagment/gameDataStructure/globalRoomDataStructure.py

```diff
@@ -1,4 +1,71 @@
 """
 This Module contains data-structures and functions that will be
 require to manage the global variable for the given gameState.
-"""
\ No newline at end of file
+"""
+from gameManagment.cardLogic.Card import generateRandom52cards
+
+import gameManagment.enums.stateConstants as stateConstant
+import gameManagment.enums.cardConstants  as cardsConstant
+
+
+class GlobalRoomDataStructure:
+    def __init__(self):
+        """
+        #TODO(Abhishek) Write definition of all variables.
+        """
+        self.allConnectedPlayers = []
+        self.cardRemainingToDistribute = generateRandom52cards()
+        self.cardThrowingStartAt = stateConstant.NO_FIXED_STARTING_PLAYER
+        self.currentGameState = stateConstant.STATE_SET_TO_CARD_DISTRIBUTION
+        self.highestBid = 0
+        self.isBidOver = False
+        self.mainPlayer = stateConstant.MAIN_PLAYER_IS_NOT_ASSIGNED
+        self.maximumCardForGivenRound = 0
+        self.playerAvailableForBid = [0, 1, 2, 3, 4, 5]
```

Comment:
what is Its Role

Reply:
playerAvailableForBid not required

Reply:
This variable was there to determine which which player can bid and which player can not